config.schema.json

```
@@ -47,6 +47,20 @@
         "required": false,
         "description": "If dimming is a valid action on a device, it will be included as additional switch on the accessory. Since this is an odd solution to dimming, it's off by default."
       },
+      "request_timeout": {
+        "title": "Timeout for requests to the bond bridge.",
+        "type": "number",
+        "default": 1000,
```

to maintain the existing values, you'll need to change this to `10000`
```suggestion
        "default": 10000,
```